### DISCUSSION TITLE
How to load a train model into PaddleOCR application?

### DISCUSSION THREAD

**Discussion by ciberola** [PaddlePaddle/PaddleOCR#6788] [Q&A] [answered]
Hi, I am training a recgnition model, English language, version: en_PP-OCRv3_rec_slim. I also have very simple Python program that is using PaddleOCR. My question is how to "load" my new trained model into my existing program? For what I have now in loading model:
ocr_model = PaddleOCR(use_angle_cls = True, use_space_char = True, lang = "en") 
English default model is downloaded, but I would like to load mine, previoulsy trained model in the above line.
Can anyone help? 
Thanks.

**Comment by LDOUBLEV** [comment, marked as answer]
U can set rec_model_dir as your trained model path. 
PaddleOCR(use_angle_cls = True, use_space_char = True, lang = "en", rec_model_dir="your trained model path")
https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.5/doc/doc_en/whl_en.md#5-parameter-description
